### ISSUE TITLE
Support onResumeCommand for restarting processes tha

### ISSUE THREAD

**Issue by nikmd23** [MicrosoftDocs/vsonline#403]
When my environment spins up for the first time, I'm able to use `postCreateCommand` to run any processes (like starting a DB) that I want. It's a great experience if I do say so myself.

However, if I'm working in an environment over the period of a few days so that the environment becomes suspended (or I self-suspend to switch instanceTypes myself), then the experience isn't as great. 😕

Unfortunately, my processes are killed and I have to remember to go into the terminal and get them back up and running. 

It'd be great if there was an `onResumeCommand`, similar to `postCreateCommand` that I could specify to automate this - keeping the experience symmetrical whether I'm creating of resuming. 

+@Chuxel to think though VS Code Remote implications.

**Comment by Chuxel** [comment]
I raised an issue in vscode-remote-release on this since I've hit a few more scenarios where I could use it. We can think through the right name - `postAttachConfig` might be an option.

**Comment by chrmarti** [comment]
With Remote-Containers there can be multiple windows connected to the same container. `onResumeCommand` might fit better if the command should only be run once for all windows, whereas `postAttachCommand` might work better if the command should run for each connecting window.

**Comment by Chuxel** [comment]
That's a good point.  "Resume" though is a bit confusing for the first time you start the container.  Perhaps `postStartCommand`?

**Comment by lostintangent** [comment]
@Chuxel @edgonmsft @jkeech Is adding support for `postStartCommand` still on our radar?

**Comment by jkeech** [comment]
> @Chuxel @edgonmsft @jkeech Is adding support for `postStartCommand` still on our radar?

What's should the priority of it be? It's not on the upcoming sprint plan.

**Comment by lostintangent** [comment]
@jkeech It's come up a few times recently when chatting with beta users. So if possible, I'd love to consider pulling it into the upcoming sprint.

**Comment by Chuxel** [comment]
@jkeech Yeah this and `${localWorkspaceFolder}` can both be blocking. I filled @edgonmsft on when this fires for Remote - Containers right now.

I'm increasingly thinking we need to start marking the devcontainer.json reference with what is supported in Codespaces given some of these gaps - people seem to be hitting them.

**Comment by mchelen-gov** [comment]
+1 for adding `postStartCommand` support because it helps automate the use case where a codespace has paused & spun down, and some actions should be run when it starts up again
